compliance_checker/protocols/cdl.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+'''
+compliance_checker/protocols/cdl.py
+'''
+import os
+
+
+def is_cdl(filename):
+    '''
+    Quick check for .cdl ascii file
+
+    Example:
+        netcdf sample_file {
+        dimensions:
+            name_strlen = 7 ;
+            time = 96 ;
+        variables:
+            float lat ;
+                lat:units = "degrees_north" ;
+                lat:standard_name = "latitude" ;
+                lat:long_name = "station latitude" ;
+        etc...
+
+    :param str filename: Absolute path of file to check
+    :param str data: First chuck of data from file to check
+    '''
+    if os.path.splitext(filename)[-1] == '.cdl':
+        return True
```

Comment:
i'm not stuck on this comment, but i would argue that checking a filename extension does not always mean it's true that it is one - i'd flip this logic and say if it doesn't end with cdl, return False, and check contents like you do below.  But like I said, I'm not hung up on this, this is certainly fine.